.env

```diff
@@ -21,7 +21,7 @@ DB_CKAN_DB=ckan
 
 # Put envvars last per envvars doc, https://github.com/okfn/ckanext-envvars#requirements
```

Comment:
Should we update this comment...?

Reply:
@robert-bryson that's my bad, I should have had you put that back after we tried flipping that around... I would put envvars last again.